srcpkgs/i3lockr/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'i3lockr'
+pkgname=i3lockr
+version=0.1.0
+revision=1
+build_style=cargo
+makedepends="rust xcb-util-image-devel python3"
+short_desc="Distort a screenshot and run i3lock"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT and APACHE"
```

Comment:
No, this is not how you declare multiple licenses. and APACHE doesn't follow spdx short-identifier